src/main/java/net/mcreator/ui/MCreatorApplication.java

```diff
@@ -71,6 +72,8 @@
 
 	private final List<MCreator> openMCreators = new ArrayList<>();
 
+	private WorkspaceSelector.RecentWorkspaceEntry recentWorkspaceEntry;

```

Comment:
Why is this needed and was not so far?

Reply:
It's maybe not needed. I added it at the beginning because the way I did, it was necessary to be able to update old workspaces, but as I changed how I made it, I maybe forgot to re-change it.

Reply:
Please cleanup the code then.